libraries/AP_GyroFFT/AP_GyroFFT.h

```diff
@@ -274,7 +282,7 @@ class AP_GyroFFT
     // engine health in tracked peaks
     uint8_t _health;
     // engine health on roll/pitch/yaw
-    Vector3<uint8_t> _rpy_health;
+    Vector3<uint32_t> _rpy_health;
```

Comment:
why this change? Should this be Vector3<bool> ?

Reply:
Ah, they should be Vector3ul. I made this change because it should save some flash - there's no good reason to use uint8_t over uint32_t and using it requires that the template methods be defined.